internal/controller/monitor/monitor.go

```diff
@@ -109,13 +110,20 @@ func (m *Monitor) Start(ctx context.Context, handler JobHandler) <-chan error {
 
 	agentTags := toMapAndLogErrors(logger, m.cfg.Tags)
 
+	var queue string
+	var ok bool
+	if queue, ok = agentTags["queue"]; !ok {
+		errs <- errors.New("missing required tag: queue")
+		return errs
+	}
```

Comment:
This is different to non-K8s agents, which default to `queue=default` https://buildkite.com/docs/agent/v3/queues#the-default-queue

But that's probably for historical reasons and I reckon it's better to be explicit.

Reply:
yeah, if this launches without a queue tag, it defaults to `queue=kubernetes`, this is just an extra layer of defense